src/Augurk/DataAccess/DocumentStoreProvider.cs

```diff
@@ -36,7 +37,7 @@ public class DocumentStoreProvider : IDocumentStoreProvider
         /// <summary>
         /// Default constructor for this class.
         /// </summary>
-        public DocumentStoreProvider(IWebHostEnvironment environment, ILogger<DocumentStoreProvider> logger)
+        public DocumentStoreProvider(IWebHostEnvironment environment, ILogger<DocumentStoreProvider> logger, ILogger<MigrationManager> migrationLogger)
```

Comment:
Waarom doen we hier niet gewoon de MigrationManager injecten?

Reply:
Kip-ei probleem. De MigrationManager heeft een IDocumentStoreProvider nodig.

Reply:
Ah, check, dat is wel lastig ja. Willen we dan misschien nog wel een regeltje commentaar opnemen om dat duidelijk te maken?

Reply:
Fixed